taskcluster/tc-package.sh

```diff
@@ -56,6 +57,7 @@ package_native_client_ndk()
   tar -cf - \
     -C ${deepspeech_dir}/native_client/libs/${arch_abi}/ deepspeech \
     -C ${deepspeech_dir}/native_client/libs/${arch_abi}/ libdeepspeech.so \
+    -C ${deepspeech_dir}/native_client/libs/${arch_abi}/ generate_scorer_package \
```

Comment:
@lissyx this isn't really doing what I expected (see corresponding changes in Android.mk). do you have any tips on how to include this binary with the package as well?